### DISCUSSION TITLE
Cannot click on the image file selector

### DISCUSSION THREAD

**Discussion by Nadia00485** [opendcim/openDCIM#1566] [Q&A]
![image](https://github.com/user-attachments/assets/e8c887b1-5e99-4f44-905d-dda17a474d67)
Hi, I cannot click on the file selector. It's like the button is unresponsive. I have already checked the code, and there seems to be nothing wrong with it. Can you provide me with a solution?

**Comment by wilpig** [comment]
Need steps to reproduce.  I have tested this on https://dev.opendcim.org/device_templates.php?TemplateID=290 and I am able to click and select images just fine.
